src/containers/SignupForm/signUpHandler.ts

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+
+import AuthService from '@/services/authService';
+import { AuthCredentials, IApiError, IUserAuthResponseMessage } from '@/constants/types';
```

Comment:
why is the first letter capital? Are we using a class/component/interface? Neither, so use `camelCase` syntax.
change the filename, function, and here.